coreneuron/nrniv/nrnoptarg.cpp

```diff
@@ -160,16 +160,6 @@ int nrnopt_parse(int argc, const char* argv[]) {
         }
     }
 
-    // last the default config file
-    std::string cfg("config");
-    if (!opt->importFile(cfg.c_str(), '#')) {
-        if (nrnmpi_myid == 0)
-            std::cerr << "Info: No default arguments file provided: " << cfg << std::endl;
-    } else {
-        if (nrnmpi_myid == 0)
-            std::cout << "Read default configuration file: " << cfg << std::endl;
-    }
-
```

Comment:
@nrnhines  : we are only using option via `-read-config`. I think we can get read of this default `config` file?